src/providers/google-reverse-geocoder.php

```diff
@@ -29,7 +29,8 @@ class GoogleReverseGeocoder implements ReverseGeocoder {
   function __construct( Support\GoogleURLBuilder $url_builder = null,
     Commands\HttpGet $curl_caller = null,
     Support\JsonDecoder $geo_json_decoder = null,
-    Support\GoogleReverseGeocoderParser $geo_parser = null ) {
+    Support\GoogleReverseGeocoderParser $geo_parser = null,
+    Support\GoogleForwardGeocoderParser $geo_parser2 = null ) {
```

Comment:
geo_parser2.. really.. shorter variable names are not faster, they are just more confusing. 😕